compositor_pipeline/src/pipeline/decoder.rs

```diff
@@ -48,9 +50,18 @@ impl<Input: PipelineInput> Decoder<Input> {
         let parameters = input.decoder_parameters();
 
         std::thread::spawn(move || {
+            let mut h264_depayloader = rtp::codecs::h264::H264Packet::default();
```

Comment:
I don't like the fact that we are using here anything from rtp package. Packets that we receive here should already in a format that does not depend on any protocol

Reply:
I want to address this in the following PRs, so I'm leaving this as-is for now.